shaptools/netweaver.py

```diff
@@ -243,7 +243,12 @@ def install(
                 cwd=' SAPINST_CWD={}'.format(cwd) if cwd else '')
         result = shell.execute_cmd(cmd, root_user, password, remote_host)
         if result.returncode and raise_exception:
-            raise NetweaverError('SAP Netweaver installation failed')
+            if cwd:
+                raise NetweaverError('SAP Netweaver installation failed. Please check swpm installation logs' \
```

Comment:
@MalloZup These lines have more than 100 characters. We have the limit in 100. Please, split them more to maximum 100 characters.

Reply:
ok will do later on thx!